go/common/logging.go

```diff
@@ -49,6 +49,12 @@ func LogTXExecution(txHash common.Hash, msg string, args ...interface{}) {
 	log.Info(txExecPattern, txHash.Hex(), formattedMsg)
 }
 
+// DebugTXExecution - logs at DEBUG level with the tx hash prepended
+func DebugTXExecution(txHash common.Hash, msg string, args ...interface{}) {
```

Comment:
what's up with this? 
If not used, pls remove

Reply:
Yep 👍  I decided last minute just bump this to log level.